### DISCUSSION TITLE
PnP search Result WebPart(v4.1) - Add support to specify which managed properties to use for hithighlighting as modern pages index the full page JSON leading to guids showing

### DISCUSSION THREAD

**Discussion by ShariqSiddiquie** [microsoft-search/pnp-modern-search#3014] [Ideas]
These are the steps. Please see attached file with screenshots.

1.SharePoint Communication site collection.
2. Create news post page, add page properties webpart and save as a page template.
3. Create another news post page using the page template (which is created in step#2) and add content (less than 20 characters) using text webpart .

PnP Search Result Page:
Issue – Highlighted content shows the GUID of the page properties web part.
Actually, It was an issue in Search Index and Microsoft fixed this issue, but this issue still coming in PnP search result webpart.

Could you please fix this issue in PnP Search webpart?
 
Please see screenshots in attached file.

Version used -  4.1

[PnPSearch- HighlightContentIssue.docx](https://github.com/microsoft-search/pnp-modern-search/files/6394909/PnPSearch-.HighlightContentIssue.docx)

**Comment by wobba** [comment]
We don't expose in the web part to decide which properties to generate a hithighlighted summary from. I'll tag this as an improvement, as it's not a web part issue. And please keep the issue template headings.. easier to digest. The templates are there for a reason.

**Comment by ShariqSiddiquie** [comment]
Thank You!

When we can expect fix this enhancement and new release package.

**Comment by wobba** [comment]
Depends on time availability, so no current plans.
